src/modules/weather/models/weather.model.ts

```diff
@@ -0,0 +1,110 @@
+import prisma from '@/config/client';
+import { handlePrismaError } from '@/errors';
+import type {
+  WeatherData,
+  CreateWeatherData,
+  UpdateWeatherData,
+} from '../types';
+
+const normalizeCreateData = (data: CreateWeatherData) => {
+  return {
+    ...data,
+  };
+};
+
+const normalizeUpdateData = (data: UpdateWeatherData) => {
+  return {
+    ...data,
+  };
+};
```

Comment:
I think, you can just do the destructing in actual model instead of two helper utils, but it works fine. just suggesting.